src/menus/optionsMenu.cpp

```diff
@@ -158,6 +161,18 @@ OptionsMenu::OptionsMenu()
         PreferencesManager::set("operations_radar_lock", value ? "1" : "");
     }))->setValue(PreferencesManager::get("science_radar_lock", "0") == "1")->setSize(GuiElement::GuiSizeMax, 50);
 
+    // Language.
+    (new GuiLabel(interface_page, "LANGUAGE", tr("Language"), 30))->addBackground()->setSize(GuiElement::GuiSizeMax, 50);
+    language_selection = new GuiSelector(interface_page, "LANGUAGE_SELECTION", [](int index, string value)
+    {
+        PreferencesManager::set("language", string(value));
+        i18n::reset();
+        i18n::load("locale/" + PreferencesManager::get("language", "en") + ".po");
+    });
+    language_selection->setOptions({"cz", "de", "en", "fr", "it"})->setSize(GuiElement::GuiSizeMax, 50);
```

Comment:
https://github.com/daid/SeriousProton/blob/master/src/resources.h#L39
The `findResources("locale/*.po")` should be able to list all locales.

Reply:
List is now populated from locale files that match most common two- or three-letter locale shortname patterns with two-letter extensions (en.po, en_uk.po, zzz_AA.po, etc.).